models/things.js

```diff
@@ -116,7 +116,7 @@ var Things = {
 
      // Notify each open websocket of the new Thing
      this.websockets.forEach(function(socket) {
-       socket.send(JSON.stringify(thing.getDescription()));
+       socket.send(JSON.stringify(thing));
```

Comment:
Both NewThingsController.get('/') and NewThingsController.ws('/') send directly JSON-ed things (https://github.com/moziot/gateway/blob/master/controllers/new_things_controller.js#L24 and https://github.com/moziot/gateway/blob/master/controllers/new_things_controller.js#L42). I didn't realize this was not the intended behavior and can modify the PR accordingly.

The problem I'm trying to solve here is that the Add New Thing UI needs to have some id for the Thing it is trying to save for the `POST /things` to work. This works in the normal cases of NewThingsController.get('/') and NewThingsController.ws('/') because they send the full Thing object for all Things currently in the database. Where it doesn't work is when a Thing is added with the Add New Thing UI open. In this case, Things.handleNewThing will be invoked and will call getDescription before sending the Thing object. Neither of the other instances of sending Thing objects calls getDescription.

This matters because if the Philips Hue is paired while the Add New Thing UI is open it will add a Thing for each bulb. The adding of these Things is communicated to the UI through  handleNewThing which invokes getDescription.